relayer-cli/src/commands/query/channels.rs

```diff
@@ -284,6 +341,27 @@ impl Debug for QueryChannelsOutput {
         match self {
             QueryChannelsOutput::Verbose(output) => write!(f, "{:#?}", output),
             QueryChannelsOutput::Summary(output) => write!(f, "{:#?}", output),
+            QueryChannelsOutput::Pretty(output) => {
+                output.iter().try_for_each(|pretty_print| {
+                    write!(
+                        f,
+                        "\n{}: {}/{} --- {}: {}/{}",
```

Comment:
To make things slightly prettier, let's use a unicode arrow instead of `---`?

For instance `⟷`

```
ibc-1: transfer/channel-0 ⟷ ibc-0: transfer/channel-0
ibc-1: transfer/channel-1 ⟷ ibc-2: transfer/channel-1
```

Reply:
Unicode arrows are pretty small on my console : 
![image](https://user-images.githubusercontent.com/92268577/180985437-8be8ecba-578e-4744-ae7c-baafc3102838.png)

Reply:
Then let's stick with ASCII!

Reply:
Wonder why your terminal does that though? Perhaps it's the font?

Reply:
I have no idea. I could push a new version with the double arrow if you want to try with your terminal as well.

Reply:
Interesting, they also render funny in iTerm on macOS.

Reply:
Maybe we can use another emoji ?